vllm/entrypoints/openai/serving_engine.py

```diff
@@ -769,6 +801,37 @@ def create_streaming_error_response(
         )
         return json_str
 
+    def _handle_error_finish_reason(
```

Comment:
wdyt about different name?
```suggestion
    def _raise_if_error(
```